pkg/discovery/kubernetes.go

```diff
@@ -65,7 +65,7 @@ func (k *KubernetesDiscovery) GetDiscoveredPlugins(clusterClient cluster.Client)
 		if err != nil {
 			logMsg += err.Error()
 		}
-		log.Debug(logMsg)
+		log.Info(logMsg)
```

Comment:
This is potentially not on an error path (for clusters that don't have the CLIPLugin CRD) and will show all the time now